src/controller/CommissioningDelegate.h

```diff
@@ -975,6 +894,7 @@ class CommissioningDelegate
     virtual void SetOperationalCredentialsDelegate(OperationalCredentialsDelegate * operationalCredentialsDelegate) = 0;
     virtual CHIP_ERROR StartCommissioning(DeviceCommissioner * commissioner, CommissioneeDeviceProxy * proxy)       = 0;
     virtual CHIP_ERROR CommissioningStepFinished(CHIP_ERROR err, CommissioningReport report)                        = 0;
+    virtual void CleanupCommissioning()                                                                             = 0;
```

Comment:
Why is this on the commissioning delegate API?  It's not called from "outside" anywhere; it's an implementation detail of AutoCommissioner.

Reply:
Removed CleanupCommissioning from the CommissioningDelegate and made virtual in the AutoCommissioner.